### ISSUE TITLE
Allow downgrade or delay auto-updates

### ISSUE THREAD

**Issue by Amoki** [ottomated/CrewLink#551]
# Pre-Flight Checklist
Please use this checklist to avoid spamming:

1. [x] I am not asking a question => use the [Discord](https://discord.gg/9mwuVNA) if you are
2. [x] I have checked that someone else has not suggested this using the [search bar](https://github.com/ottomated/CrewLink/issues?q=is%3Aissue)
3. [x] My feature request is not one of the **commonly suggested features**:
- Adjustable voice radius
- Mobile support
- Linux / MacOS support
- Itch.io support

**Is your feature request related to a problem? Please describe.**
I'm playing with friends on linux. Linux forks are not yet on v2.0 (still in 1.2.1).
Windows launcher auto updates CrewLink to 2.0.1 so I can't play with them.
I didn't find any way to force CrewLink to stay at 1.2.1

**Describe the solution you'd like**
Add an option to disable auto updates

**Comment by billyjbryant** [comment]
I agree that upgrades should be an opt-in process wherein the user is prompted that an upgrade is available, especially when moving from a minor to major release as you do moving from 1.2.1 to 2.0.x.  The prompt should provide the user with the latest release notes (or a link where they can view them) and the option to choose whether or not they wish to upgrade at this time. 

In addition, I have found the auto upgrade to be clunky in that it often fails and ultimately gives up and stays reverted on my system. This is not an ideal UX.

**Comment by Classist** [comment]
A lot of people can't use crewlink since 2.0.1 also because #501 #534, this feature would be great.

**Comment by duracell** [comment]
I created a version without auto-update:
https://github.com/duracell/CrewLink-NoAutoUpdate/releases

**Comment by billyjbryant** [comment]
@duracell, that's fine, however, I think the priority really should be to fix auto update.
